src/Components/QuestionComponentContainer/QuestionComponentContainer.js

```diff
@@ -175,56 +177,70 @@ const QuestionComponentContainer = ({
   };
 
   const renderHeaderAndSubheader = () => {
-    if (matchingQuestion.headerType === 'signUpInfo') {
-      return (
-        <h1 className="sub-header">
-          <FormattedMessage
-            id="qcc.optional-sign-up-text"
-            defaultMessage="Optional: Sign up for benefits updates and/or paid feedback opportunities"
-          />
-        </h1>
-      );
-    } else if (matchingQuestion.headerType === 'aboutHousehold') {
-      if (matchingQuestion.name === 'hasBenefits' || matchingQuestion.name === 'acuteHHConditions') {
+    const { headerType, name } = matchingQuestion;
+
+    switch (headerType) {
+      case 'signUpInfo':
         return (
           <h1 className="sub-header">
             <FormattedMessage
-              id="qcc.tell-us-final-text"
-              defaultMessage="Tell us some final information about your household."
+              id="qcc.optional-sign-up-text"
+              defaultMessage="Optional: Sign up for benefits updates and/or paid feedback opportunities"
             />
           </h1>
         );
-      } else if (matchingQuestion.name === 'referralSource') {
-        return (
-          <h1 className="sub-header">
-            <FormattedMessage id="questions.referralSource" defaultMessage="Just one more question!" />
-          </h1>
-        );
-      } else {
+      case 'aboutHousehold':
+        if (name === 'hasBenefits' || name === 'acuteHHConditions') {
+          return (
+            <h1 className="sub-header">
+              <FormattedMessage
+                id="qcc.tell-us-final-text"
+                defaultMessage="Tell us some final information about your household."
+              />
+            </h1>
+          );
+        } else if (name === 'zipcode') {
+          return (
+            <>
+              <h2 className="question-secondary-header">Let&#39;s Get Started!</h2>
```

Comment:
All of our text is wrapped in a `FormattedMessage` so that it can be translated.

Reply:
I've updated this. Thanks!